in_toto/runlib.py

```diff
@@ -818,11 +843,21 @@ def in_toto_record_stop(step_name, product_list, signing_key=None,
     unfinished_fn = UNFINISHED_FILENAME_FORMAT.format(step_name=step_name,
         keyid=signing_key["keyid"])
 
+    if metadata_directory is not None:
+      unfinished_fn = os.path.join(metadata_directory, unfinished_fn)
+
   # FIXME: Currently there is no way to know the default GPG key's keyid and
   # so we glob for preliminary link files
+  # FIXME: UNFINISHED_FILENAME_FORMAT_GLOB_DIR has a problem with window system
```

Comment:
Did you get a chance to confirm this? :)

Reply:
I found an IBM [QA page](https://www.ibm.com/support/pages/windows-forward-slash-and-backslash-characters-separators) which say that "windows can use the windows directory separator (\) and the Unix directory separator(/)" However, I didn't find the Microsoft documentation that confirms this.

Reply:
> Did you get a chance to confirm this? :)

I found an IBM [QA page](https://www.ibm.com/support/pages/windows-forward-slash-and-backslash-characters-separators) which says that "windows can use the windows directory separator(\) and the Unix directory separator(/)" However, I didn't find the Microsoft documentation that confirms this.